backend/src/main/java/de/armbrust/planz/amazonapi/AmazonApiLogic.java

```diff
@@ -0,0 +1,32 @@
+package de.armbrust.planz.amazonapi;
+
+import com.amazon.sellingpartner.api.ReportsApi;
+import com.amazon.sellingpartner.model.ReportDocument;
+import com.amazon.spapi.documents.DownloadBundle;
+import de.armbrust.planz.model.Product;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+public class AmazonApiLogic {
```

Comment:
maybe a bit picky, but I'd leave away or replace "Logic" from the class name. Otherwise you would need to name almost all your classes like this